### DISCUSSION TITLE
Can't calculate wavelength accurately

### DISCUSSION THREAD

**Discussion by joesmerdon** [capytaine/capytaine#470] [Q&A] [answered]
Hi,

I'm specifying the angular frequency of the waves in my simulations. When I calculate the wavelength using 2pi.g/omega^2, it's off by a percent or so, increasing as omega increases. I'm using the best current value for g, 9.80665. 

Am I missing anything? Please tell me even if it's glaringly obvious, because I am new to this.

Thanks

Joe

**Comment by mancellin** [comment, marked as answer]
That's the right formula in infinite depth (not in finite depth).
But Capytaine uses $g=9.81$ by default, could this be the issue?

**Reply by joesmerdon** [reply]
To get my calculation to fit, I need to vary g between 9.74 at omega=3 to 9.0 at omega=8, so I think it's something else.
I do have the water depth set to infinite.

**Reply by mancellin** [reply]
Could you share a piece of code for me to investigate?

**Reply by joesmerdon** [reply]
Thanks for your responses.  On further investigation, it appears that there is no problem. I was measuring something wrong.
